docs/EDGEVIEW-CONTAINER-API.md

```diff
@@ -23,3 +23,69 @@ For security reasons, the Edge-View container has all the volumes mounted in 're
 ## Logging to Event
 
 Edge-View logging is similar to other containers. For any user command received by Edge-View, it will log the client endpoint(IP address/port), the command and its parameters. The log entry will also be tagged with object-type of `log-to-event`, and the controller can optionally process those log entries and generate them as device events or alerts.
+
+## Edge-View Client Authentication
+
+To address limitations in controlling and tracking user access to Edge-View commands, the authentication enhancements to Edge-View aim to strengthen user authentication, enforce stricter access controls, and improve user tracking.
```

Comment:
"enhancement" sounds like you are describing a set of changes, but the documentation here should describe how it works post the changes in the PR.

Reply:
True. will modify.